### PR TITLE
Removed TermError from the UX

### PR DESCRIPTION
**Pull request by GeorgebigG**
Removed TermError from the School form UX (from the form and table for course). You can check by going to Schools then go to either create school or edit school and also you can see from the table that the Term Error has been removed

http://organic-george2.dokku-09.cs.ucsb.edu/

### CODE REVIEW

**Review by JUstISAGE — DISMISSED** [review]
LGTM

File: frontend/src/tests/components/School/SchoolForm.test.js [hunk 1]
```diff
@@ -64,7 +64,7 @@ describe("SchoolForm tests", () => {
         expect(screen.getByText(/Name is required./)).toBeInTheDocument();
         expect(screen.getByText(/Term Regex is required./)).toBeInTheDocument();
         expect(screen.getByText(/Term Description is required./)).toBeInTheDocument();
-        expect(screen.getByText(/Term Error is required./)).toBeInTheDocument();
+        // expect(screen.getByText(/Term Error is required./)).toBeInTheDocument();
```
Comment (pconrad) [inline frontend/src/tests/components/School/SchoolForm.test.js]: Please remove commented out code when merging into main.
See: https://ucsb-cs156.github.io/topics/code_reviews/commented_out_code.html

File: frontend/src/tests/components/School/SchoolForm.test.js [hunk 2]
```diff
@@ -80,14 +80,14 @@ describe("SchoolForm tests", () => {
         const nameField = screen.getByTestId("SchoolForm-name");
         const termRegexField = screen.getByTestId("SchoolForm-termRegex");
         const termDescriptionField = screen.getByTestId("SchoolForm-termDescription");
-        const termErrorField = screen.getByTestId("SchoolForm-termError");
+        // const termErrorField = screen.getByTestId("SchoolForm-termError");
```
Comment (pconrad) [inline frontend/src/tests/components/School/SchoolForm.test.js]: Please remove commented out code when merging into main.
See: https://ucsb-cs156.github.io/topics/code_reviews/commented_out_code.html

File: frontend/src/tests/components/School/SchoolForm.test.js [hunk 3]
```diff
@@ -80,14 +80,14 @@ describe("SchoolForm tests", () => {
...
         const submitButton = screen.getByTestId("SchoolForm-submit");
 
         fireEvent.change(abbrevField, { target: { value: 'ucsb' } });
         fireEvent.change(nameField, { target: { value: 'UC Santa Barbara' } });
         fireEvent.change(termRegexField, { target: { value: '[WSMF]\\d\\d' } });
         fireEvent.change(termDescriptionField, { target: { value: 'Enter quarter, e.g. F23, W24, S24, M24' } });
-        fireEvent.change(termErrorField, { target: { value: 'Quarter must be entered in the correct format' } });
+        // fireEvent.change(termErrorField, { target: { value: 'Quarter must be entered in the correct format' } });
```
Comment (pconrad) [inline frontend/src/tests/components/School/SchoolForm.test.js]: Please remove commented out code when merging into main.
See: https://ucsb-cs156.github.io/topics/code_reviews/commented_out_code.html

File: frontend/src/tests/components/School/SchoolForm.test.js [hunk 4]
```diff
@@ -96,7 +96,7 @@ describe("SchoolForm tests", () => {
         expect(screen.queryByText(/Name is required./)).not.toBeInTheDocument();
         expect(screen.queryByText(/Term Regex is required./)).not.toBeInTheDocument();
         expect(screen.queryByText(/Term Description is required./)).not.toBeInTheDocument();
-        expect(screen.queryByText(/Term Error is required./)).not.toBeInTheDocument();
+        // expect(screen.queryByText(/Term Error is required./)).not.toBeInTheDocument();
```
Comment (pconrad) [inline frontend/src/tests/components/School/SchoolForm.test.js]: Please remove commented out code when merging into main.
See: https://ucsb-cs156.github.io/topics/code_reviews/commented_out_code.html

File: frontend/src/tests/pages/SchoolCreatePage.test.js [hunk 5]
```diff
@@ -62,7 +62,7 @@ describe("SchoolCreatePage tests", () => {
             name: "name-1",
             termRegex: "[WSMF]\\d\\d",
             termDescription: "F23",
-            termError: "error-1"
+            // termError: "error-1"
```
Comment (pconrad) [inline frontend/src/tests/pages/SchoolCreatePage.test.js]: Please remove commented out code when merging into main.
See: https://ucsb-cs156.github.io/topics/code_reviews/commented_out_code.html

File: frontend/src/tests/pages/SchoolCreatePage.test.js [hunk 6]
```diff
@@ -82,15 +82,15 @@ describe("SchoolCreatePage tests", () => {
         const nameField = screen.getByTestId("SchoolForm-name");
         const termRegexField = screen.getByTestId("SchoolForm-termRegex");
         const termDescriptionField = screen.getByTestId("SchoolForm-termDescription");
-        const termErrorField = screen.getByTestId("SchoolForm-termError");
+        // const termErrorField = screen.getByTestId("SchoolForm-termError");
```
Comment (pconrad) [inline frontend/src/tests/pages/SchoolCreatePage.test.js]: Please remove commented out code when merging into main.
See: https://ucsb-cs156.github.io/topics/code_reviews/commented_out_code.html

File: frontend/src/tests/pages/SchoolCreatePage.test.js [hunk 7]
```diff
@@ -82,15 +82,15 @@ describe("SchoolCreatePage tests", () => {
...
         const submitButton = screen.getByTestId("SchoolForm-submit");
 
        
         fireEvent.change(abbrevField, { target: { value: 'abb' } });
         fireEvent.change(nameField, { target: { value: 'name-1' } });
         fireEvent.change(termRegexField, { target: { value: '[WSMF]\\d\\d' } });
         fireEvent.change(termDescriptionField, { target: { value: 'F23' } });
-        fireEvent.change(termErrorField, { target: { value: 'error-1' } });
+        // fireEvent.change(termErrorField, { target: { value: 'error-1' } });
```
Comment (pconrad) [inline frontend/src/tests/pages/SchoolCreatePage.test.js]: Please remove commented out code when merging into main.
See: https://ucsb-cs156.github.io/topics/code_reviews/commented_out_code.html

File: frontend/src/tests/pages/SchoolCreatePage.test.js [hunk 8]
```diff
@@ -103,7 +103,7 @@ describe("SchoolCreatePage tests", () => {
                 "name": "name-1",
                 "termRegex": "[WSMF]\\d\\d",
                 "termDescription": "F23",
-                "termError": "error-1"
+                // "termError": "error-1"
```
Comment (pconrad) [inline frontend/src/tests/pages/SchoolCreatePage.test.js]: Please remove commented out code when merging into main.
See: https://ucsb-cs156.github.io/topics/code_reviews/commented_out_code.html

File: frontend/src/tests/pages/SchoolEditPage.test.js [hunk 9]
```diff
@@ -107,7 +107,7 @@ describe("SchoolEditPage tests", () => {
             const nameField = screen.getByTestId("SchoolForm-name");
             const termRegexField = screen.getByTestId("SchoolForm-termRegex");
             const termDescriptionField = screen.getByTestId("SchoolForm-termDescription");
-            const termErrorField = screen.getByTestId("SchoolForm-termError");
+            // const termErrorField = screen.getByTestId("SchoolForm-termError");
```
Comment (pconrad) [inline frontend/src/tests/pages/SchoolEditPage.test.js]: Please remove commented out code when merging into main.
See: https://ucsb-cs156.github.io/topics/code_reviews/commented_out_code.html

File: frontend/src/tests/pages/SchoolEditPage.test.js [hunk 10]
```diff
@@ -118,15 +118,15 @@ describe("SchoolEditPage tests", () => {
             expect(termRegexField).toHaveValue("reg-17");
             expect(termDescriptionField).toBeInTheDocument();
             expect(termDescriptionField).toHaveValue("description-17");
-            expect(termErrorField).toBeInTheDocument();
-            expect(termErrorField).toHaveValue("error-17");
+            // expect(termErrorField).toBeInTheDocument();
+            // expect(termErrorField).toHaveValue("error-17");
```
Comment (pconrad) [inline frontend/src/tests/pages/SchoolEditPage.test.js]: Please remove commented out code when merging into main.
See: https://ucsb-cs156.github.io/topics/code_reviews/commented_out_code.html

File: frontend/src/tests/pages/SchoolEditPage.test.js [hunk 11]
```diff
@@ -118,15 +118,15 @@ describe("SchoolEditPage tests", () => {
...
 
             expect(submitButton).toHaveTextContent("Update");
 
             fireEvent.change(nameField, { target: { value: 'new-name-17' } });
             fireEvent.change(termRegexField, { target: { value: 'new-reg-17' } });
             fireEvent.change(termDescriptionField, { target: { value: 'new-description-17' } });
-            fireEvent.change(termErrorField, { target: { value: 'new-error-17' } });
+            // fireEvent.change(termErrorField, { target: { value: 'new-error-17' } });
```
Comment (pconrad) [inline frontend/src/tests/pages/SchoolEditPage.test.js]: Please remove commented out code when merging into main.
See: https://ucsb-cs156.github.io/topics/code_reviews/commented_out_code.html

File: frontend/src/tests/pages/SchoolEditPage.test.js [hunk 12]
```diff
@@ -161,20 +161,20 @@ describe("SchoolEditPage tests", () => {
             const nameField = screen.getByTestId("SchoolForm-name");
             const termRegexField = screen.getByTestId("SchoolForm-termRegex");
             const termDescriptionField = screen.getByTestId("SchoolForm-termDescription");
-            const termErrorField = screen.getByTestId("SchoolForm-termError");
+            // const termErrorField = screen.getByTestId("SchoolForm-termError");
```
Comment (pconrad) [inline frontend/src/tests/pages/SchoolEditPage.test.js]: Please remove commented out code when merging into main.
See: https://ucsb-cs156.github.io/topics/code_reviews/commented_out_code.html

File: frontend/src/tests/pages/SchoolEditPage.test.js [hunk 13]
```diff
@@ -161,20 +161,20 @@ describe("SchoolEditPage tests", () => {
...
             const submitButton = screen.getByTestId("SchoolForm-submit");
 
     
             expect(abbrevField).toHaveValue("abb");
             expect(nameField).toHaveValue("name-17");
             expect(termRegexField).toHaveValue("reg-17");
             expect(termDescriptionField).toHaveValue("description-17");
-            expect(termErrorField).toHaveValue("error-17");
+            // expect(termErrorField).toHaveValue("error-17");
```
Comment (pconrad) [inline frontend/src/tests/pages/SchoolEditPage.test.js]: Please remove commented out code when merging into main.
See: https://ucsb-cs156.github.io/topics/code_reviews/commented_out_code.html

File: frontend/src/tests/pages/SchoolEditPage.test.js [hunk 14]
```diff
@@ -161,20 +161,20 @@ describe("SchoolEditPage tests", () => {
...
 
             fireEvent.change(nameField, { target: { value: 'new-name-17' } });
             fireEvent.change(termRegexField, { target: { value: 'new-reg-17' } });
             fireEvent.change(termDescriptionField, { target: { value: 'new-description-17' } });
-            fireEvent.change(termErrorField, { target: { value: 'new-error-17' } });
+            // fireEvent.change(termErrorField, { target: { value: 'new-error-17' } });
```
Comment (pconrad) [inline frontend/src/tests/pages/SchoolEditPage.test.js]: Please remove commented out code when merging into main.
See: https://ucsb-cs156.github.io/topics/code_reviews/commented_out_code.html

**Review by pconrad — CHANGES_REQUESTED** [review]
Please remove commented out code when merging into main.
See: https://ucsb-cs156.github.io/topics/code_reviews/commented_out_code.html

**Review by Peak1260 — APPROVED** [review]
Looks like the unnecessary comments were taken out and everything looks good

**Review by pconrad — APPROVED** [review]
LGTM

**Review by pconrad — APPROVED** [review]
LGTM

**Pull request merged**